### ISSUE TITLE
Help with IE9 Compatability

### ISSUE THREAD

**Issue by mgfranz** [malsup/corner#53]
This code works fine with FF and other versions of IE, but with IE9 it does not round.  First I test version and then set parameters;

<script type="text/javascript">
if (/MSIE (\d+\.\d+);/.test(navigator.userAgent)){ //test for MSIE x.x;
 var ieversion=new Number(RegExp.$1) // capture x.x portion and store as a number
 if (ieversion <=6)
    $(document).ready(function(){
    $("div.rounded").corner('top 10px')
    });
 else if (ieversion>=7)
    $(document).ready(function(){
    $("div.rounded").corner();
    $(".content").corner('bottom 10px');
    }); 
}
 else
    $(document).ready(function(){
    $("div.rounded").corner();
    $(".header").corner('top 10px')
    $(".content").corner('bottom 10px');
});

</script>

Then I use the 'div' tags to control layout;

```
//<div class="rounded" style="background:#BFB28D; padding:1px;">
                //<div class="header" style="padding:5px; white-space:nowrap; background-image: url(../Templates/images/red.jpg);">  Videos </div>
                //<div class="content"> Links to other stuff here... </div>
```

So basically I use a background image in the "header" div, but in IE9 the div's are square.  Link to test page, http://www.mgfic.com/default.aspx
